### ISSUE TITLE
Embedded browser doesn't open

### ISSUE THREAD

**Issue by ashishbinu** [antfu/vscode-vite#8]
After clicking the `vite` button at bottom status bar, I clicked on open with embedded browser and it showed error as shown in image below: 

<img src="https://user-images.githubusercontent.com/56691622/119155717-174e2480-ba71-11eb-8266-250b91c5f510.png" height="500px"/>

**Comment by antfu** [comment]
Have you installed Chrome?

**Comment by ashishbinu** [comment]
@antfu I have chrome installed. 
![image](https://user-images.githubusercontent.com/56691622/119182094-1ed0f600-ba90-11eb-9328-6f8349441fe3.png)

**Comment by CreatorB** [comment]
> @antfu I have chrome installed.
> ![image](https://user-images.githubusercontent.com/56691622/119182094-1ed0f600-ba90-11eb-9328-6f8349441fe3.png)

Hi, @ashishbinu I've got the same issue here, but already solved by manually add a chrome path on the extension setting, see attached screenshot below.

![image](https://user-images.githubusercontent.com/6529730/120911486-0842b580-c6b2-11eb-8852-beedb9872895.png)

**Comment by ashishbinu** [comment]
Yeah , It worked but shouldn't it work without giving the path. Whoever is gonna install it in linux might not have it working when they run it.

**Comment by antfu** [comment]
If you find a way to cover your cases to do the auto-detection, I am more than happy to receive contributions to that.

https://github.com/antfu/vscode-browse-lite/blob/2995249e7332675ff3e719a8b0a9567cfcb7dd39/src/BrowserClient.ts#L69-L82
